src/components/RockerSwitch.vue

```diff
@@ -21,6 +21,18 @@
 </template>
 
 <script>
+import colorString from 'color-string';
+
+export const isColorValid = (color) => {
```

Comment:
@vyzaldysanchez I prefer to extract this function to a helper file (`src/utils/helpers.js`) and just import it here!

Reply:
Done @DannyFeliz, I was thinking about it but couldn't decide if calling the folder `utils` o creating a validators file.

Reply:
I normally use this name because most of the time we don't validate stuff like this manually, we use a lib or something. In `util` folder I tent to put my global filters, helpers and stuff like that so just to keep it simple we can let it like that.

Reply:
@vyzaldysanchez BTW, can you get the double quote back and remove the trailing comma 👀?

Reply:
The double quote is back and I removed the trailing commas. @DannyFeliz 

We should define then better `eslint` so it yells about having single quotes and trailing commas.